lstchain/reco/reconstructorCC.py

```diff
@@ -227,6 +227,69 @@ def template_interpolation(gain, times, t0, dt, a_hg, a_lg):
     return out
 
 
+@njit(cache=True)
+def nsb_only_waveforms(nsb_waveform, time, is_high_gain, additional_nsb, amplitude, t_0,
+                           t0_template, dt_template, a_hg_template, a_lg_template):
+    """
+    Generate waveforms of pure NSB. NSB photons injected through a fast interpolator using a
+    uniformly sampled base with known origin and step with a certain amplitude at t_0.
```

Comment:
Generate waveforms of pure NSB -> technically add pure NSB to a waveform since you take a waveform as input. 

I see two way to change the code (current version is also good, simply ideas). 
1: Keep the function the same but pass the waveform to be modified instead of creating an intermediate zeros waveform. (function could then be "add_nsb_only_waveform") -> advantage : no intermediate waveform object and removes the final sum out of the function
2: Create the zeros waveform in the function -> advantage : removes input and guaranties that the output is indeed an NSB only waveform. (get n_pixels from "additional_nsb") -> possible problem : call to numpy possible and as fast in this case?

Reply:
I would prefer option 2. Maybe it is more clean than option 1 and the the overall time of the function `tune_nsb_on_waveform` should be the same.